examples/event.py

```diff
@@ -0,0 +1,28 @@
+# Registering Events
+
+from EpikCord import Client, Intents, Messageable
+
+intents = Intents().all()
```

Comment:
This is supposed to be `Intents.all()`
Unless ya'll changed intents code, but last time i changed it was to inherit my Flag class.

Reply:
???